docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -114,4 +116,125 @@ If `false`, {es} returns an error if the `path` is an unmapped field.
 
 You can use this parameter to query multiple indices that may not contain the
 field `path`.
---
\ No newline at end of file
+--
+
+[[nested-query-notes]]
+==== Notes
+
+[[multi-level-nested-query-ex]]
+===== Multi-level nested queries
+
+To see how multi-level nested queries work,
+try the following example.
+
+First, create an index, `drivers`,
+with a nested field mapping.
+
+[source,console]
+----
+PUT /drivers
+{
+    "mappings" : {
+        "properties" : {
+            "driver" : {
+                "type" : "nested",
+                "properties" : {
+                    "vehicle" : {
+                        "type" : "nested",
+                        "properties" : {
+                            "model" : {                 <1>
+                                "type" : "keyword"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+----
+
+<1> Note the `model` field is nested as follows:
++
+```
+driver → vehicle → model
+```
+
+Next, index a document to the `drivers` index.
+
+[source,console]
+----
+PUT /drivers/_doc/1?refresh
+{
+  "driver" : {
+        "vehicle" : {
+            "model" : "Canyonero"
+        }
+    }
+}
+----
+// TEST[continued]
+
+You can now use a search request with a multi-level nested query
+to return documents based on the `model` field.
```

Comment:
I think I'd just say, "You can now use a multi-level nested query..." Instead of saying that you use a query "to return documents", maybe "...to match documents based on the `model` field"? Generally, it's the _reqest_ that returns the response. The query identifies what you're looking for, it doesn't do the returning.  You could say that the user does something to _get_ documents, but saying that they do it to return docs is odd.